dotnet/samples/GettingStartedWithProcesses/Step01/Step01_Processes.cs

```diff
@@ -64,6 +66,15 @@ public async Task UseSimpleProcessAsync()
         // Build the process to get a handle that can be started
         KernelProcess kernelProcess = process.Build();
 
+        // Generate a Mermaid diagram for the process and print it to the console
+        string mermaidGraph = kernelProcess.ToMermaid();
+        Console.WriteLine($"=== Start - Mermaid Diagram for '{process.Name}' ===");
+        Console.WriteLine(mermaidGraph);
+        Console.WriteLine($"=== End - Mermaid Diagram for '{process.Name}' ===");
+
+        // Generate an image from the Mermaid diagram
+        await MermaidRenderer.GenerateMermaidImageAsync(mermaidGraph, "ChatBotProcess.png");
```

Comment:
perhaps add a console write line specifying - `$"Process Mermaid Image generated: {filepath}$`. Since this is in the samples and the image gets dumped in the assembly dir users when running this may not even notice an image was generated.

Also wondering if this function should return the absolute filepath of the image

Reply:
I think it makes sense to get the absolute filepath... so the user/dev can find it...

Reply:
updated & demo showing a console writeline :)